license/usage-logs/main.go

```diff
@@ -0,0 +1,81 @@
+/*
+ * This example showcases how to see license usage logs
+ * in docx to PDF conversing with UniOffice.
+ */
+
+package main
+
+import (
+	"fmt"
+	"log"
+	"os"
+	"path"
+	"path/filepath"
+	"strings"
+
+	unipdflicense "github.com/unidoc/unipdf/v3/common/license"
+
+	"github.com/unidoc/unioffice/common/license"
+	"github.com/unidoc/unioffice/common/logger"
+	"github.com/unidoc/unioffice/document"
+	"github.com/unidoc/unioffice/document/convert"
+)
+
+func init() {
+	// Make sure to load your metered License API key prior to using the library.
+	// If you need a key, you can sign up and create a free one at https://cloud.unidoc.io
+	err := unipdflicense.SetMeteredKey(os.Getenv(`UNIDOC_LICENSE_API_KEY`))
+	if err != nil {
+		fmt.Printf("ERROR: Failed to set metered key: %v\n", err)
+		fmt.Printf("Make sure to get a valid key from https://cloud.unidoc.io\n")
+		fmt.Printf("If you don't have one - Grab one in the Free Tier at https://cloud.unidoc.io\n")
+		panic(err)
+	}
```

Comment:
Probably can make a simple operation for this example?
like open and then modify docx, so we can remove this code and we also already had convert to pdf example.